drivers/spi/spi_nxp_lpspi/Kconfig

```diff
@@ -1,26 +1,41 @@
-# MCUXpresso SDK SPI
-
-# Copyright (c) 2018, NXP
+# Copyright 2018, 2024 NXP
 # SPDX-License-Identifier: Apache-2.0
 
 config SPI_MCUX_LPSPI
-	bool "MCUX LPSPI driver"
+	bool "NXP LPSPI peripheral"
```

Comment:
Suce you change mcux to nxp, shouldn't this change be made on other places where relevant? (Iike the option for dma support etc...)

Reply:
I kept MCUX on the things that still use the mcux branded hal intentionally.

As for the driver vs peripheral word in the prompt, I'd prefer to say driver since it's the driver software that the kconfig enables